src/bytecode/flags.py

```diff
@@ -42,6 +42,34 @@ class CompilerFlags(IntFlag):
         FUTURE_ANNOTATIONS = 0x1000000
 
 
+def ignored_instruction(i) -> bool:
+    return isinstance(
+        i,
+        (
+            _bytecode.SetLineno,
+            _bytecode.Label,
+            _bytecode.TryBegin,
+            _bytecode.TryEnd,
+        ),
+    )
+
+
+def is_generator(instructions) -> bool:
+    # Look for a YIELD_VALUE that is not for an await
+    found_yield = False
+    for i in instructions:
+        if ignored_instruction(i):
+            continue
+        if found_yield:
+            if i.name == "RESUME" and i.arg == 3:
```

Comment:
On 3.13 you need to focus on the 2 lowest bits. The third bit encode another piece of information.

Reply:
Updated now